packages/protocol-kit/src/adapters/web3/Web3Adapter.ts

```diff
@@ -32,6 +32,7 @@ import {
   getSignMessageLibContractInstance,
   getSimulateTxAccessorContractInstance
 } from './contracts/contractInstancesWeb3'
+import { SigningMethod } from '../..'
```

Comment:
Could we set a route here that is easier to understand?

`@safe-global/protocol-kit` or `@safe-global/protocol-kit/types`